sdks/python/apache_beam/runners/portability/fn_api_runner_transforms.py

```diff
@@ -1018,8 +1018,8 @@ def fuse(producer, consumer):
       for output in transform.outputs.values():
         producers_by_pcoll[output] = stage
 
-  logging.debug('consumers\n%s', consumers_by_pcoll)
-  logging.debug('producers\n%s', producers_by_pcoll)
+  #logging.debug('consumers\n%s', consumers_by_pcoll)
+  #logging.debug('producers\n%s', producers_by_pcoll)
```

Comment:
I commented these out because they are very noisy and don't provide much value. Someone debugging them can uncomment them, but otherwise, I think it makes sense to not have them.

Reply:
Logging.debug is fairly low level, but if it's too verbose for even that then go ahead and delete these lines rather than comment them out.

Reply:
What happens here is that when a test fails in `tox`, debug logging gets activated, and we currently log almost every proto related to portability, so it's a ton of logging, and it's very hard to read.

When running the tests with `python -m unittest module.TestClass.test_case`, this is not a problem. Debug logging does not get activated on failure, so it's easier to read through. 

It would be nice to have a middle ground - but I lean towards removing them until someone finds them useful in debugging, and we can putthem back. Thoughts?

Reply:
Vote for deleting them. If a dev needs these info, the dev can add these logs.

Reply:
Done.

Reply:
Yes, delete. The value of these logs is low now that this stuff is basically working (and it'd be good to fix the tox issue, but we shouldn't wait for that.)